### PR TITLE
RIP `Current`

### PR DESCRIPTION
**Pull request by stephencelis**
(Branched off #810.)

### CODE REVIEW

**Comment by stephencelis** [comment]
🫡

File: Sources/PointFree/Host.swift [hunk 1]
```diff
@@ -1,26 +1,29 @@
+import Dependencies
 import Views
 
 extension Host {
   static var brandon: Host {
+    @Dependency(\..assets.brandonImgSrc) var brandonImgSrc
```
Comment (mbrandonw) [inline Sources/PointFree/Host.swift:6]: huh somehow this is compiling??

Comment (mbrandonw) [reply Sources/PointFree/Host.swift:6]:
```suggestion
    @Dependency(\.assets.brandonImgSrc) var brandonImgSrc
```

File: Sources/PointFree/Host.swift [hunk 2]
```diff
@@ -1,26 +1,29 @@
...
-      image: Current.assets.brandonImgSrc,
+      image: brandonImgSrc,
       name: "Brandon Williams",
       twitterRoute: .mbrandonw,
       website: "http://www.fewbutripe.com"
     )
   }
 
   static var stephen: Host {
+    @Dependency(\..assets.stephenImgSrc) var stephenImgSrc
```
Comment (mbrandonw) [inline Sources/PointFree/Host.swift:20]:
```suggestion
    @Dependency(\.assets.stephenImgSrc) var stephenImgSrc
```

File: Tests/PointFreeTests/PrivateRssTests.swift [hunk 3]
```diff
@@ -42,6 +42,7 @@ class PrivateRssTests: TestCase {
     user.rssSalt = "deadbeef"
 
     await withDependencies {
+      $0.calendar = .init(identifier: .gregorian)
```
Comment (mbrandonw) [inline Tests/PointFreeTests/PrivateRssTests.swift:45]: worth overriding this once in `invokeTest`? or not worth it?

**Review by mbrandonw — APPROVED** [review]

**Pull request merged**